publications-2019.bib

```diff
@@ -1,5 +1,31 @@
 % Encoding: US-ASCII
 
+@article{Heron2018,
```

Comment:
```suggestion
@article{Heron2019,
```

Reply:
Let's not worry about this and save the time and energy to change it -- we ignore these tags anyway when building the website.